src/applications/veteran-id-card/containers/EmailCapture.jsx

```diff
@@ -53,12 +52,12 @@ class EmailCapture extends React.Component {
             <a href="/privacy-policy/">See our privacy policy</a>
           </p>
           <form onSubmit={this.handleSubmit}>
-            <TextInput
-              errorMessage={this.props.errors && this.props.errors[0].title}
-              label={<span>Email address</span>}
+            <va-text-input
+              error={this.props.errors && this.props.errors[0].title}
+              label="Email address"
               name="email"
-              field={this.props.email}
-              onValueChange={update => this.props.setEmail(update)}
+              value={this.props.email}
+              onInput={e => this.props.setEmail(e.target.value)}
```

Comment:
I'm going to pause on changes for this PR and try to have a quick & safe (_but still not great_ - migration will need to happen eventually) solution coming from the `component-library` instead. The reason is because there are no tests to cover changes like these which is concerning:

![No tests passing or failing in veteran-id-card application](https://user-images.githubusercontent.com/2008881/171730346-05ba0e1f-714f-4c50-9bb0-dd7dc738ed47.png)